### PR TITLE
Demo updates

### PR DESCRIPTION
**Pull request by kellyhutchins**
Add new demos

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -177,7 +176,8 @@ Web Content Accessibility Guidelines
 <!-- .slide: data-background="images/background/01.svg" -->
 
 <p class="centerp">
-    <a href="https://developers.arcgis.com/javascript/latest/sample-code/widgets-basemapgallery/live/index.html">Demo: Text alternative</a>
+    <a href="https://developers.arcgis.com/javascript/latest/sample-code/widgets-basemapgallery/live/index.html">Demo: Text alternative - empty alt</a>
+     <a href="https://story.maps.arcgis.com/apps/MapJournal/index.html?appid=2b1c793f464b4cd2944a0b9700c0dc48">Demo: Text alternative - descriptive</a>
```
Comment (jimmieego) [inline index.md]: We need a line break here, or we can use `ul` to list the two demos.

File: demos/color-contrast/index.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                "Percent change is {POP2013:populationChange}"
+            };
+
+            layer.popupTemplate = popupTemplate;
+
+            populationChange = function (value, key, data) {
+              // calculate the population percent change from 2010 to 2013.
+              var diff = data.POP2013 - data.POP2010;
+              var pctChange = (diff * 100) / data.POP2010;
+              var result = diff > 0 ? "up.png" : "down.png";
```
Comment (jimmieego) [inline demos/color-contrast/index.html:93]: It looks like the up and down images in the popups are broken?

File: index.md (hunk 1 above)
Comment (kellyhutchins) [reply index.md]: I'll switch to ul

File: demos/color-contrast/index.html (hunk 2 above)
Comment (kellyhutchins) [reply demos/color-contrast/index.html:93]: Yep looks like i forgot to copy over the images to the demos folder. Good catch

**Comment by jimmieego** [comment]
👍 
merging

**Pull request merged**